src/main/kotlin/fi/metatavu/muisti/geometry/GeometryTransformer.kt

```diff
@@ -0,0 +1,88 @@
+package fi.metatavu.muisti.geometry
+
+import com.vividsolutions.jts.geom.Coordinate
+import com.vividsolutions.jts.geom.GeometryFactory
+import com.vividsolutions.jts.geom.Point
+import fi.metatavu.muisti.api.spec.model.Bounds
+import fi.metatavu.muisti.api.spec.model.Coordinates
+import fi.metatavu.muisti.api.spec.model.Polygon
+
+/**
+ * Converts spec Polygon to Geometry Polygon
+ *
+ * @param polygon spec polygon
+ * @return null or Geometry Polygon
+ */
+fun getPolygon(polygon: Polygon?): com.vividsolutions.jts.geom.Polygon? {
+
+  polygon ?: return null
+
+  val geometryFactory = GeometryFactory()
+  val coordinates = mutableListOf<Coordinate>()
+  polygon.shapes.forEach { shape ->
+    shape.forEach {coordinate ->
+      coordinates.add(Coordinate(coordinate[0], coordinate[1]))
+    }
+  }
+
+  return geometryFactory.createPolygon(coordinates.toTypedArray())
+}
+
+/**
+ * Convert Geometry Polygon from entity to spec Polygon.
+ * Note: Since each room can only have one shape we can hardcode the first index to 0.
```

Comment:
does this note actually belong to this method? seems a bit specific for generic function